osCam/userconfig/views.py

```diff
@@ -1,111 +1,231 @@
-from ipaddress import ip_address
-from django.http import JsonResponse
-from django.shortcuts import render
-import json as Json
-
-# from core import models
-from userconfig.models import Network, Camera, RaspberryPi
 from django.contrib.auth.models import User
-# from userconfig.models import Network
-
-################################
-################################
-#  Views used for User Configurations: .
-#       e.g. A Config is of Type::{RaspberryPi, Camera, Network}
-def configPage(request):
-    '''
-        View Handles Http response To Display all user defined Configuration.
-            Configuration Could be of Type::{RaspberryPi, Camera, Network}.
-        
-        @param  request: Request 
-    '''
-    page = "User Configuration Page"
+from django.shortcuts import render, redirect
+from userconfig.forms import *
+from userconfig.models import *
+
+def settings(request):
+
+    from userconfig.models import Camera
+
+    this_user = User.objects.get(username=request.user.username)
+
+    from userconfig.models import Network
+    # if there is no Network settings, create
+    if (Network.objects.count() < 1):
+        Network.objects.create(
+            user=this_user,
+            homeIpAddress = '192.168.0.69',
+            homeNetmask = '255.255.0.0',
+            cameraIpAddress = '192.168.0.70',
+        )
+
+    # if there are no pis, create a starter pi 
+    if (RaspberryPi.objects.count() < 1):
+        RaspberryPi.objects.create(
+            user = this_user,
+            modelName = 'model name',
+            modelNum = '1234',
+            username = 'username',
```

Comment:
I not sure about some of these default configs

Reply:
well for sure they're bad lol but idk how many of them are even necessary so didnt want to spend too much time on it before reconsidering the variables we really need access to / want users to have access to / need to use somewhere in our code. this was mostly just scaffolding.